src/scripts/variables.js

```diff
@@ -0,0 +1,17 @@
+export const httpMethods = [
+  "GET",
+  "POST",
+  "PUT",
+  "PATCH",
+  "DELETE",
+  "COPY",
+  "HEAD",
+  "OPTIONS",
+  "LINK",
+  "UNLINK",
+  "PURGE",
+  "LOCK",
+  "UNLOCK",
+  "PROPFIND",
+  "VIEW",
+];
```

Comment:
this is cool, but remember that we use one branch for one feature, well git history is really important

Reply:
clear, I will add this in next PR,